src/provider/providerWrapper.ts

```diff
@@ -61,6 +61,15 @@ export class ProviderWrapper {
   }
 
   async getErc725YVersion(address: string): Promise<ERC725_VERSION> {
+    const isErc725YBatch = await this.supportsInterface(
+      address,
+      ERC725Y_INTERFACE_IDS['5.0'],
+    );
+
+    if (isErc725YBatch) {
+      return ERC725_VERSION.ERC725_BATCH;
+    }
```

Comment:
similar to my comment above, i would stick to version number instead of features.